libs/expo/betterangels/src/lib/static/constants.ts

```diff
@@ -6,13 +6,20 @@ export const ServicesByCategory = [
     items: [
       ServiceEnum.Food,
       ServiceEnum.Water,
+      ServiceEnum.Bag,
```

Comment:
Hi @mikefeldberg 

I noticed a few things.

1. **Socks - Clothes or Clothes - Socks**
In the spreadsheet, it shows "Clothes - Socks" - it should be consistent. 

2. **The new items from Harm Reduction are missing.**
- Harm Reduction - Safe Smoking
- Harm Reduction - Narcan
- Harm Reduction - Test Kit

3. **Search is not fully functioning**
When the items are with multiple words such as Sleeping Bag, or HMIS Consent, when I typed the first word, I see it in the search result. However, when I fully typed the name in, the search result shows "No Results".

FYI: I only tested this on iOS. I am still having the same issue with Android (I need to reach out Paul for this).

Reply:
1. updated
2. we decided to remove these for now, since we already have "harm reduction"
3. this is the same behavior as in main

Reply:
Ok it looks good. Approved.